src/lib/styles/foundation/partyColors.scss

```diff
@@ -614,3 +614,9 @@
         background-color: var(--undeclared) !important;
     }
 }
+
+img {
+    &.before-color--lab {
```

Comment:
would simply `.before-color--lab` work here or is it not specific enough?

Reply:
deleted this - it's now being handled in the component itself which is better